packages/components/src/components/forms/PasswordInput.tsx

```diff
@@ -174,7 +174,7 @@ export class PasswordInput extends React.Component<
       <StyledField>
         <StyledInput
           ref={this.$element}
-          {...this.props}
+          {...(this.props as any)}
```

Comment:
This is needed to suppress the typing error. Maybe the Storybook update also installs a newer version of `@types/react`, which clashes a bit with the old Typescript version we have 🤔

Reply:
This was fixed.